src/main/java/org/springframework/integration/aws/inbound/SqsMessageDrivenChannelAdapter.java

```diff
@@ -88,6 +89,10 @@ public void setResourceIdResolver(ResourceIdResolver resourceIdResolver) {
 		this.simpleMessageListenerContainerFactory.setResourceIdResolver(resourceIdResolver);
 	}
 
+	public void setAutoStartUp(boolean autoStartUp) {
+		super.setAutoStartup(autoStartUp);
```

Comment:
Not enough. We have to have your 
```
this.simpleMessageListenerContainerFactory.setAutoStartup(autoStartUp);
```
here any way.
The test should pass only basing on the XML config. Where indeed that `auto-startup="false"` has to be delegated to the `simpleMessageListenerContainerFactory`.